Automattic/jetpack
# Boost: Add annotations support to performance history graph

## Proposed changes:
Changing how scores are calculated will confuse users. So, we need a way to communicate this change. This PR adds annotations support to performance history graph to add annotations for this and anything else.


https://github.com/Automattic/jetpack/assets/3737780/8b6a90ca-13d4-4269-85f0-c06f2ad65576



### Other information:


## Jetpack product discussion
pc9hqz-2rd-p2#comment-1910

## Does this pull request change what data or activity we track or use?



## Testing instructions:
* Make sure performance history still loads.
* Try [injecting some annotations](https://github.com/Automattic/jetpack/blob/77648dc719780925765aa89d004a3d68ad5bc7a4/projects/plugins/boost/app/data-sync/Performance_History_Entry.php#L34) with performance history data in the following format:

```php
'annotations' => [
	[
		'timestamp' => 1704067200000,
		'text'      => 'Happy New Year!',
	]
]
```
